docs/api/catalog-resource.md

```diff
@@ -257,6 +258,16 @@ The `published` timestamp is the time when the package was last listed.
 > [!Note]
 > On nuget.org, the `published` value is set to the year 1900 when the package is unlisted.
 
+#### Vulnerabilities
+
+An array of `vulnerability` objects. Each vulnerability has the following properties:
+
+Name         | Type   | Required | Notes
+------------ | ------ | -------- | -----
+advisoryUrl  | string | yes      | Location of security advisory for the package
+gitHubDatabaseKey | string | yes      | GitHub reference for security advisory
+severity     | string | yes      | Severity of advisory: "0" = Low, "1" = Moderate, "2" = High, "3" = Critical
```

Comment:
How should one interpret an unknown severity string? This is useful if we want to extend this severity protocol in the future

Reply:
I've not seen any advisories with severity == unknown. Is this a possibility?

Reply:
I’m not aware of any other values today. However, how should clients that we release today behave if we update the protocol tomorrow with values the client won’t understand? We document the expected client behavior in the protocol docs (for example, we document how client should behave on I know deprecation reason or unknown property)

Reply:
It's a good question. I would lean toward making unknown behave as Moderate, however this behavior hasn't been spec'd. Opinions? @anangaur @joelverhagen @loic-sharma

Reply:
Actually, looking at the philosophy employed in deprecation reasons (which is to ignore unknowns), I'm changing my thinking to "treat as Low" for unknowns here.